Bundle/Command/ImportSubscribers.php

```diff
@@ -136,6 +136,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        ini_set('memory_limit', -1);
```

Comment:
I think -1 is not good, eventually add a option to override it if necessary but not with -1